tests/unit/splinter_test.c

```diff
@@ -115,9 +115,8 @@ CTEST_DATA(splinter)
 // clang-format off
 CTEST_SETUP(splinter)
 {
-   if (Ctest_verbose) {
-      platform_set_log_streams(stdout, stderr);
-   }
+   set_log_streams_for_tests();
```

Comment:
For symmetry, I introduced this `set_` method, too. This is really not needed, but it makes it easier for new unit-test dev to just follow one `set_log_streams_for*()` pattern.